database/migrations/2022_12_10_183251_increment_user_i_ds.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up(): void
+	{
+		Schema::disableForeignKeyConstraints();
+		/** @var App\Models\User|null $admin */
+		$admin = DB::table('users')->find(0);
+		if ($admin !== null && ($admin->username === '' || $admin->password === '')) {
+			// The admin user (id 0) has never set a username and password, so we remove it.
+			// This should only happen on a completely new installation where the admin user is created by the
+			// MigrateAdminUser migration and the user has never logged in.
+			DB::table('users')->delete(0);
```

Comment:
```suggestion
			DB::table('users')->where('id', '=', 0)->delete();
```
The result is the same, but it is more explicit of what is going on.